admin/src/components/Tables/StatisticTable.vue

```diff
@@ -0,0 +1,91 @@
+<!-- eslint-disable vue/no-static-inline-styles -->
+<template>
+  <div class="statistic-table">
+    <b-table-simple
+      style="width: auto"
+      class="mt-5"
+      :items="items"
+      :fields="fields"
```

Comment:
What is this? Never defined

Reply:
Thank you for the hint.
They're removed by https://github.com/gradido/gradido/commit/7ef66e9da4219f4726b92aa6583b7b4ff4ec9527.